packages/taiko-client/bindings/encoding/struct.go

```diff
@@ -41,6 +41,19 @@ type BlockParams struct {
 	Signature      []byte
 }
 
+// BlockParams2 should be same with TaikoData.BlockParams2.
+type BlockParams2 struct {
+	Coinbase           common.Address
+	ExtraData          [32]byte
+	ParentMetaHash     [32]byte
+	AnchorBlockId      uint64
+	Timestamp          uint64
+	BlobTxListOffset   uint32
+	BlobTxListLength   uint32
+	BlobIndex          uint8
+	BasefeeSharingPctg uint8
```

Comment:
No `BasefeeSharingPctg` in latest version
```suggestion
```

Reply:
Removed